playground.py

```diff
@@ -0,0 +1,282 @@
+from typing import final
```

Comment:
I noticed that you have copied and pasted all your functions in your playground files. By including `from viewing-party.party import *` you can import all your functions and simply test them in the playground files with data. Copying code can lead to bugs.

In addition, you may consider not comiting these files to github.